calculus/functions.md

```diff
@@ -62,6 +62,99 @@
 :::
 ::::
 
+## Evaluating and Manipulating Trigonometric Functions
+```{image} ./_images/MAtappunitcircle66.png
+:alt: Unit Circle
+:width: 300px
+:align: center
+```
+ Each quadrant of the unit circle corresponds to a positive trigonometric function as represented by "A(all)+", "T(tangent)+", "S(sine)+", "C(cosine)+".
+
+The given unit circle has a triangle drawn within which represents how trigonometric functions are related to lengths and heights of the sides of right traingles. 
+The circle has a radius and is centered at (0,0). A point on the circle is defined by angle $\theta$ measured counterclockwise from the positive x-axis.
+The trigonometric functions describe how the coordinates of a point on the unit circle relate to the angle $\theta$.
+\begin{align}
+\sin(\theta)& = \frac{y}{r}& 
+\cos(\theta)& = \frac{x}{r}& 
+\tan(\theta)& = \frac{y}{x}\\
+\csc(\theta)& = \frac{r}{y}&
+\sec(\theta)& = \frac{r}{x}&
+\cot(\theta)& = \frac{x}{y}\\
+\end{align}
+
+Angles can be measured in degrees of radians
+\begin{align}
+\theta(rad.)& = \theta(deg.)(\frac{\pi}{180^{\circ}})
+\end{align}
+
+### Special Angles
+\begin{align}
+\theta& = 30^{\circ} \frac{\pi}{6}& \theta& = 60^{\circ} \frac{\pi}{3}
+\end{align}
+
+```{image} ./_images/306090triangle.png
+:alt: 30 60 90 Traingle
+:width: 300px
+:align: center
+```
+
+At $ \theta = 30^{\circ} $:
+\begin{align}
+\sin(\theta)& = \frac{1}{2}& 
+\cos(\theta)& = \frac{\sqrt{3}}{2}
+\end{align}
```

Comment:
```suggestion
\begin{equation}
\sin(\theta) = \frac{1}{2} \quad \cos(\theta) = \frac{\sqrt{3}}{2}
\end{equation}
```
I will admit this is a clever (if convoluted) way to add spacing between equations that are on the same line. However, I believe using `\quad` would be more recommended in this case.